university-of-helsinki-faculty-of-theology.csl

```diff
@@ -279,34 +279,34 @@
   <macro name="publisher">
     <choose>
       <if type="bill graphic legal_case legislation motion_picture report song" match="any">
-        <group prefix=" ">
-          <group suffix="; ">
+        <group prefix=" " suffix=" " delimiter="; ">
```

Comment:
this needs a semicolon in the suffix, since in the original there was one as part of the last group.